### ISSUE TITLE
Error message escapes current configuration dialog on Linux desktop

### ISSUE THREAD

**Issue by igor2x** [dbeaver/dbeaver#17505]
### Description

DBeaver testing connection on Linux desktop. When error message is displayed and Alt+Tab clicked multiple times then error message is hidden behind dialog from where error was triggered.

### DBeaver Version

Community Edition Version 22.1.5.202208211008

### Operating System

Ubuntu 22.04

### Database and driver

Not applicable.

### Steps to reproduce

In DBeaver Community Edition 22.1.5 on Ubuntu 22.04:
1. From **Database Navigator** right click end **Edit Connection**.
2. Click on button **Test Connection** and if connection fails, then error message dialog is displayed as expected.
3. Now press Alt+Tab to intent to change program to some other program and repeat Alt+Tab to get back to DBeaver and problem appears. I expect the Error message dialog is at the top of dialogs, but it actually escapes behind _Connection configuration_ dialog and it is not really clear (if not looking carefully) why can't I click on Cancel or OK button on _Connection configuration_ dialog (because there is error dialog hidden behind current dialog).
![image](https://user-images.githubusercontent.com/10895030/187444937-293b867f-9574-44b3-9f68-d42d48178203.png)

Please compare the same behavior after **Test Connection** button is clicked and connection is successful. In this case successful message never escapes behind _Connection configuration_ dialog.
![image](https://user-images.githubusercontent.com/10895030/187444881-a3f60cbf-2424-447d-96b7-51805fec1165.png)

I have also tested DBeaver Community Edition 22.1.5 on Windows 10 and I could not reproduce the problem. It looks like the Linux specific problem.

**Comment by Destrolaric** [comment]
Looks similar to https://github.com/dbeaver/dbeaver/issues/13691

**Comment by igor2x** [comment]
Yes, it looks a duplicate. Maybe leave it open, because I have added additional info that error messages escape main window, but successful messages don't. Maybe there is some setting about this two windows that can be helpful for developers.
